cmd/root.go

```diff
@@ -56,16 +61,23 @@ func NewRootCmd() *cobra.Command {
 					fmt.Sprintf("net.netfilter.nf_conntrack_tcp_timeout_close_wait=%d", options.TimeoutCloseWaitSecs),
 				)
 				out, err := sysctl.CombinedOutput()
-				log.Println(string(out))
 				if err != nil {
+					log.Errorln(string(out))
 					return err
+				} else {
+					log.Println(string(out))
```

Comment:
rather than using the `Print` functions (which I believe log at Info level), let's be explicit about the level and instead use something like `log.Info`.